### PR TITLE
Fix CTV for GPU

### PR DESCRIPTION
**Pull request by marchdf**

### CODE REVIEW

**Comment by marchdf** [comment]
I tested this on an Eagle GPU node and seems to runs just fine.

File: amr-wind/physics/ConvectingTaylorVortex.cpp [hunk 1]
```diff
@@ -161,8 +162,10 @@ amrex::Real ConvectingTaylorVortex::compute_error(
 
 void ConvectingTaylorVortex::output_error()
 {
-    const amrex::Real u_err = compute_error(0, m_velocity, u_exact);
-    const amrex::Real v_err = compute_error(1, m_velocity, v_exact);
+    UExact u_exact;
+    VExact v_exact;
```
Comment (sayerhs) [inline amr-wind/physics/ConvectingTaylorVortex.cpp:166]: you don't need these. You can remove the first and third argument from `compute_error` and just do 

```C++
compute_error<UExact>(m_velocity);
```

Comment (marchdf) [reply amr-wind/physics/ConvectingTaylorVortex.cpp:166]: let me try that!

Comment (marchdf) [reply amr-wind/physics/ConvectingTaylorVortex.cpp]: Ok I fixed that. Thanks for seeing that

**Pull request merged**